Aggiunto capitolo CV per un developer (#112)
Il-Libro-Open-Source/book: docs/il-curriculum/README.md

Nel CV di esempio, chiaramente, le soft skills inserite sono volutamente goliardiche (come quasi tutto il resto del CV d’altronde). Ma riguardo questa sezione mi piacerebbe che tu facessi un piccolo “esercizio mentale” e trovassi le soft skill che, a tuo avviso, ti caratterizzano davvero. Possono essere un ottimo spunto di conversazione con chi ti valuterà!

9) Corsi e Certificazioni
Molte aziende, come requisito obbligatorio per alcune posizioni, richiedono delle certificazioni specifiche. Di conseguenza, questa sezione è importante proprio per segnalare di aver completato determinati corsi o avere in mano delle certificazioni riconosciute, spesso anche a livello globale. È inoltre una sezione utile per dimostrare il tuo impegno personale e la volontà di accrescere le tue competenze tecniche.

> Capita spesso, ad esempio, di trovare nei CV corsi e certificazioni sullo Scum/Agile/PM che possono essere usate come spunto di conversazione: spesso denotano una volontà di voler crescere professionalmente e quindi un segnale del tipo: “hey vedi, ho una certificazione da PM, è quello che voglio fare ‘da grande’”!

Reviewer: ```suggestion
Capita spesso, ad esempio, di trovare nei CV corsi e certificazioni sullo Scrum/Agile/PM che possono essere usate come spunto di conversazione: spesso denotano una volontà di voler crescere professionalmente e quindi un segnale del tipo: “hey vedi, ho una certificazione da PM, è quello che voglio fare ‘da grande’”!
```

10) Lingue e Hobby

> Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! E' invece una delle sezioni più interessanti (ed in alcuni casi, divertente) di tutto il CV. Ti permette di farti conoscere come come persona, e non solo come tecnico. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.

Reviewer: ```suggestion
Il 90% dei CV che normalmente i/le recruiter ricevono NON ha la sezione hobby, ed è un peccato! È invece una delle sezioni più interessanti (ed in alcuni casi, divertenti) di tutto il CV. Ti permette di farti conoscere come persona, e non solo come tecnico. Fornisce inoltre un sacco di spunti di conversazione utili a carpire ulteriori punti di vista, modi di vedere le cose, obiettivi raggiunti e molto altro.
```

Ad esempio, una persona che pratica sport estremi avrà probabilmente una sana propensione a superare i propri limiti, la curiosità di provare cose nuove e sicuramente coraggio. Una persona che pratica sport di squadra invece sarà sicuramente abituata a lavorare in team, a collaborare e tenderà a non essere un “lupo solitario”. Gli hobby raccontano tantissimo di te, inseriscili nel CV! Riguardo le lingue poco da dire: siamo nel 2023, è FONDAMENTALE quanto meno conoscere un’altra lingua (preferibilmente l’inglese) oltre all’italiano. Anche in questo caso, eventuali certificazioni sulle lingue, se le hai, inseriscile nel CV in questa sezione.

11) L’Autorizzazione al Trattamento Dati
Il Decreto Legislativo 10 agosto 2018, n. 101 ha stabilito che il consenso al trattamento dei dati personali presenti nel CV non è più dovuto. Questo significa che in base alla nuova normativa sulla privacy, non è obbligatorio includere il consenso al trattamento dei dati nel Curriculum Vitae. Ciascun candidato può scegliere liberamente di inserirlo o meno. Detto questo, consiglio comunque di inserire la frase che puoi tranquillamente copiare dal CV di esempio per diversi motivi:

> Inserire l’Autorizzazione al Trattamento dei Dati Personali nel proprio Curriculum Vitae significa autorizzare chi riceve il CV (il datore di lavoro, i responsabili Risorse Umane e chi si occupa della selezione) al trattamento dei dati coperti e tutelati dalla legge sulla Privacy italiana e dal GDPR – Regolamento Generale sulla Protezione Dati, in vigore in tutti i paesi dell’Unione Europea dal 25 maggio 2018

Reviewer: ```suggestion
- Inserire l’Autorizzazione al Trattamento dei Dati Personali nel proprio Curriculum Vitae significa autorizzare chi riceve il CV (il datore di lavoro, i responsabili Risorse Umane e chi si occupa della selezione) al trattamento dei dati coperti e tutelati dalla legge sulla Privacy italiana e dal GDPR – Regolamento Generale sulla Protezione Dati, in vigore in tutti i paesi dell’Unione Europea dal 25 maggio 2018
```

> Pur non essendo obbligatorio inserire l’autorizzazione al trattamento dei dati nel CV, è utile farlo perché si permette immediatamente al datore di lavoro di usare i propri dati personali per essere contattati per un colloquio oppure di conservarli per future selezioni

Reviewer: ```suggestion
- Pur non essendo obbligatorio inserire l’autorizzazione al trattamento dei dati nel CV, è utile farlo perché si permette immediatamente al datore di lavoro di usare i propri dati personali per essere contattati per un colloquio oppure di conservarli per future selezioni
```

> Se il datore di lavoro riceve un CV privo di autorizzazione, deve richiedere al candidato l’autorizzazione fornendo tutti i dettagli sulle modalità di utilizzo dei dati. Solo dopo aver ricevuto questo consenso potrà utilizzarli, di conseguenza come puoi ben immaginare questo porta a lungaggini nell’iter di selezione ed un ulteriore “layer” di burocrazia che si può tranquillamente evitare!

Reviewer: ```suggestion
- Se il datore di lavoro riceve un CV privo di autorizzazione, deve richiedere al candidato l’autorizzazione fornendo tutti i dettagli sulle modalità di utilizzo dei dati. Solo dopo aver ricevuto questo consenso potrà utilizzarli, di conseguenza come puoi ben immaginare questo porta a lungaggini nell’iter di selezione ed un ulteriore “layer” di burocrazia che si può tranquillamente evitare!
```

> Se stai leggendo questo capitolo probabilmente ti stai guardando attorno, alla ricerca di un nuovo lavoro, una nuova collaborazione, una nuova RAL (si spera più alta dell'attuale).

Reviewer: ```suggestion
# Il Curriculum

Se stai leggendo questo capitolo probabilmente ti stai guardando attorno, alla ricerca di un nuovo lavoro, una nuova collaborazione, una nuova RAL (si spera più alta dell'attuale).
```
